recipes/stemflow/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "stemflow" %}
+{% set version = "0.0.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 194af6f1247875eb7e0eadbec327f29e29d569484337cc45e003b7a565315814
+
+build:
+  number: 0
+  skip: true  # [win, osx]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - setuptools_scm
+    - setuptools
+    - geopandas >=0.11.1
+    - h3pandas >=0.2.3
+    - matplotlib-base >=3.7.1
+    - numpy >=1.24.3
+    - pandas >=2.0.3
+    - pip
+    - python
+    - scikit-learn >=1.2.2
+    - seaborn >=0.11.2
+    - tqdm >=4.65.0
+    - xgboost >=2.0.0
+    - watermark >=2.4.3
+  run:
+    - geopandas >=0.11.1
+    - h3pandas >=0.2.3
+    - matplotlib-base >=3.7.1
+    - numpy >=1.24.3
+    - pandas >=2.0.3
+    - python
+    - scikit-learn >=1.2.2
+    - seaborn >=0.11.2
+    - tqdm >=4.65.0
+    - xgboost >=2.0.0
+    - watermark >=2.4.3
+test:
```

Comment:
probably a good idea to add a `pip check` test:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L61-L64

Reply:
Thanks for the comment. `pip check` added.